.github/README.md

```diff
@@ -1,50 +1,50 @@
 
-## Why Open Source?
-As an Open Source platform, Umbraco is more than just a CMS. We are transparent with our roadmap for future versions, our incremental sprint planning notes are publicly accessible, and community contributions and packages are available for all to use.
+## Contributing
 
-## Trying out Umbraco CMS
-
-[Umbraco Cloud](https://umbraco.com/cloud) is the easiest and fastest way to use Umbraco yet, with full support for all your custom .NET code and integrations. You're up and running in less than a minute, and your life will be made easier with automated upgrades and a built-in deployment engine. We offer a free 14-day trial, no credit card needed.
-
-If you want to DIY, you can [download Umbraco](https://our.umbraco.com/download) either as a ZIP file or via NuGet. It's the same version of Umbraco CMS that powers Umbraco Cloud, but you'll need to find a place to host it yourself, and handling deployments and upgrades will be all up to you.
-
-## Community
+Umbraco is contribution-focused and community-driven. If you want to contribute back to Umbraco, please check out our [guide to contributing](CONTRIBUTING.md).
 
-Our friendly community is available 24/7 at the community hub we call ["Our Umbraco"](https://our.umbraco.com). Our Umbraco features forums for questions and answers, documentation, downloadable plugins for Umbraco, and a rich collection of community resources.
+## Join the Umbraco community
 
-## Contribute to Umbraco
+Our friendly community is available 24/7 at the community hub we call ["Our Umbraco"](https://our.umbraco.com/). Our Umbraco features forums for questions and answers, documentation, downloadable plugins for Umbraco, and a rich collection of community resources.
 
-Umbraco is contribution-focused and community-driven. If you want to contribute back to Umbraco, please check out our [guide to contributing](CONTRIBUTING.md).
+Besides "Our", we all support each other also via:
 
-## Found a bug?
+- Twitter: [Umbraco HQ](https://twitter.com/umbraco), [Release Updates](https://twitter.com/umbracoproject), [#umbraco](https://twitter.com/hashtag/umbraco)
+- Gitter: https://gitter.im/umbraco/Umbraco-CMS
+- Slack: http://umbracians.chat/
 
-Another way you can contribute to Umbraco is by providing issue reports. For information on how to submit an issue report, refer to our [online guide for reporting issues](https://github.com/umbraco/Umbraco-CMS/blob/dev-v7/.github/CONTRIBUTING.md).
+## License
 
-You can comment and report issues on the [github issue tracker](https://github.com/umbraco/Umbraco-CMS/issues).
-Since [September 2018](https://umbraco.com/blog/a-second-take-on-umbraco-issue-tracker-hello-github-issues/), the old issue tracker is in read-only mode, but can still be found at [http://issues.umbraco.org](http://issues.umbraco.org).
+Umbraco CMS is and always will be [MIT licensed](LICENSE.md) open source software.
```

Comment:
I can't really tell whether this is one of the things that will work when this PR is merged into `dev-v7`, but I think the link to the license file is broken (both the badge and the link at the bottom).

As `README.md` is located inside the magic `.github` folder, GitHub will generate a link relative to this folder - but `LICENSE.md` is located at the root of the repository.

I've tested this be accessing your own fork here: https://github.com/mzajkowski/Umbraco-CMS/tree/temp-polished-readme

Reply:
You're right @abjerner! I've missed it. Fixing both issues now...